mart/configs/attack/object_detection_mask_adversary.yaml

```diff
@@ -1,22 +1,13 @@
 defaults:
   - adversary
-  - perturber: default
-  - perturber/initializer: constant
-  - perturber/projector: mask_range
+  - pgd
+  - mask
   - composer: overlay
-  - /optimizer@optimizer: sgd
-  - gain: rcnn_training_loss
   - gradient_modifier: sign
+  - gain: rcnn_training_loss
   - objective: zero_ap
-  - enforcer: default
-  - enforcer/constraints: [mask, pixel_range]
 
 # Make a 5-step attack for the demonstration purpose.
 max_iters: 5
-
-optimizer:
-  lr: 55
-
-perturber:
-  initializer:
-    constant: 127
```

Comment:
Where did this go?

Reply:
Restored in [9b9bba9](https://github.com/IntelLabs/MART/pull/211/commits/9b9bba925b81a901bc655e9cfc646d617a4ae566)